docs/data-sources/consul_cluster.md

```diff
@@ -49,6 +49,7 @@ data "hcp_consul_cluster" "example" {
 - **region** (String) The region where the HCP Consul cluster is located.
 - **scale** (Number) The the number of Consul server nodes in the cluster.
 - **self_link** (String) A unique URL identifying the HCP Consul cluster.
+- **size** (String) The t-shirt size representation of each server VM that this Consul cluster is provisioned with. Valid options - x_small, small, medium, large.
```

Comment:
Agreed on consistency on external messaging. Is there a link/will there be a link available with this info ? Just so that we do not have to duplicate changes to the messaging in multiple places.

Reply:
Updated based on discussion with @srikrishmurthy.